lib/tasks/web.rake

```diff
@@ -74,6 +74,33 @@ namespace :slack do
           'bin/commands.rb',
           commands_template.result(files: data.keys.map { |key| key.tr('.', '_') })
         )
+
+        ### Errors
+
+        # Use the official spec to pull a list of errors and create classes for each
+
+        spec = JSON.load(open('https://raw.githubusercontent.com/slackapi/slack-api-specs/master/web-api/slack_web_openapi_v2.json'))
+        known_typos = %w[account_inactiv token_revokedno_permission]
+
+        errors = spec['paths'].map do |_, path|
+          path.map do |_, v|
+            v['responses'].map do |_, response|
+              response.dig('schema', 'properties', 'error', 'enum')
+            end
+          end
+        end.flatten.compact.uniq.sort
+
+        errors -= known_typos
+
+        error_template = Erubis::Eruby.new(File.read('lib/slack/web/api/templates/error.erb'))
+        errors.each do |error|
+          rendered_error = error_template.result(error: error)
+          File.write("lib/slack/web/api/errors/#{error}.rb", rendered_error)
+        end
```

Comment:
I considered putting all the classes into one file, instead of each class in its own file, since there's so many of them. I went this way because it follows the existing pattern more closely, but I can switch to one file if you think that's better.

Reply:
I think 1 file is better for this since they are all 1-liners. Sort them alphabetically to have a stable list.